installation/resources/production/ory-persistance-profile.yaml.tpl

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
```

Comment:
Production profile should be a single file in installation resources where all components will have its production profiles